duckduckgo/iOS
# Brad/tracking params

Task/Issue URL: https://app.asana.com/0/72649045549333/1201286795398788/f
Tech Design URL: https://app.asana.com/0/481882893211075/1201299313936485/f
CC: @bwaresiak 

**Description**:
This PR adds support for removing URL parameters known to track users cross site.

**Steps to test this PR**:
**Note: Privacy Test Page is pending merge so will need to be tested locally**
1. Disable the experiment in `VariantManager`
1. Visit https://privacy-test-pages.glitch.me/privacy-protections/query-parameters/
2. Click the links on the page
3. The displayed "Results" on the landing page should match the "Expected" from the previous page



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
